crates/miden-lib/src/account/faucets/mod.rs

```diff
@@ -220,6 +223,24 @@ impl TryFrom<&Account> for BasicFungibleFaucet {
 // FUNGIBLE FAUCET
 // ================================================================================================
 
+pub struct FungibleFaucet;
```

Comment:
I would make this an extension trait for `Account`s because it's more convenient to use. So if this is `pub trait FungibleFaucetExt` and we implement `FungibleFaucetExt for Account`, then we could do `account.get_issuance_from_account()`.

Reply:
Good idea! Is there a reason not to call it just `FungibleFaucet`?